src/main/java/bisq/desktop/main/portfolio/editoffer/EditOpenOfferView.java

```diff
@@ -131,9 +131,9 @@ protected void deactivate() {
     ///////////////////////////////////////////////////////////////////////////////////////////
 
     public void initWithData(OpenOffer openOffer) {
+        model.initWithData(openOffer);
```

Comment:
I would prefer if we don't have that call before the super call. Carries risky dependencies of order of calls (see my comment in other commit - maybe that solves the issue without changing here the order?)